fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -3437,7 +3469,7 @@ private boolean areAllRecordTypesSince(@Nullable Collection<RecordType> recordTy
             try {
                 recordStoreStateRef.set(writeOnlyState);
                 // TODO: FDBRecordStoreBase.checkPossiblyRebuild() could take a long time if the record count index is split into many groups (https://github.com/FoundationDB/fdb-record-layer/issues/7)
```

Comment:
Will this issue remain open? Will there be another change to address it beyond giving the caller more control like here? If not, maybe the comment doesn't need to reference it.